src/ffi/datetime.rs

```diff
@@ -62,11 +70,14 @@ pub struct PyDateTime_CAPI {
     #[cfg(Py_3_7)]
     pub TimeZone_FromTimeZone:
         unsafe extern "C" fn(offset: *mut PyObject, name: *mut PyObject) -> *mut PyObject,
+
+    // Defined for PyPy as a separate symbol
```

Comment:
These comments are confusing. Separate from what?

Reply:
changed it to better reflect how to access these methods on PyPy